src/plots/layout_attributes.js

```diff
@@ -182,6 +182,7 @@ module.exports = {
         'legend': 'Legend',
         'annotations': 'Annotations',
         'shapes': 'Shapes',
+        'images': 'Images',
```

Comment:
@mdtusz can you also add `images` to this test [list](https://github.com/plotly/plotly.js/blob/19030614649428a0341d28402178bdd32fc75357/test/jasmine/tests/plotschema_test.js#L118) ?